content/en/catalyst/fund9/challenges/legal-and-finacial-implementation/campagin-brief.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "legal & finacial implementation"
+linkTitle: "legal & finacial implementation"
+type: docs
+hide_feedback: true
+description: "How can we build effective solutions to explore and provide legal & financial services and support to funded Proposers."
+---
+<!-- <img src="https://cardano.ideascale.com/community-library/accounts/93/936143/Public/21-Community-Advisor-Improvements-dbdbe2.png" style="width:500px;height500px"> -->
+
+Fund Size: **500,000 USD paid in Ada**
+
+[Campaign page](https://cardano.ideascale.com/c/idea/399771) and [browse ideas]().
+
+## Challenge Brief
+Currently the real Challenge for Funded proposals relies on implementing legal and financial boundaries and frameworks for their projects.
+Providing funded proposals with solutions and services to operate legally in their residing country will open new possibilities and provide peace of mind to the project owners.
+Many projects will broaden their horizons and increase their impact on the Ecosystem once legal and financial frameworks are explored and established.
+This will in turn promote adoption and improve product ownership. There is a need to get accountants and lawyers into the Cardano & Catalyst Ecosystem.
+
+### Why is it important?
+When receiving Funds, many emerging Projects in Catalyst face legal and financial issues which block them from moving forward.
+
+### What does success look like?
+A direct access to solutions which provide financial and legal services, support and guidance to emerging Projects in the Catalyst Ecosystem
+
+### Key Metrics to measure
```

Comment:
'Key Metrics to measure' contents should be:

Use Case - How many funded proposals are making use of these new solutions and services

Impact - How many of these new implementations are mutualistic to funded proposals.

Scalability - How scalable are the solutions introduced in this challenge. Are provided solutions able to scale to 10x 100x 1000x ?

Adaptability - Can these solutions and services be used anywhere in the world